lib/types/config.js

```diff
@@ -210,6 +210,20 @@
   * number/decimal library for values of datatype <em>Number</em>.  If not
   * specified, the driver will use Javascript <em>number</em> type to
   * represent values of type Number.  See {@link DBNumberConfig}
+  * @property {boolean} [longAsBigInt] If set to <em>true</em>, Database Type
```

Comment:
This is a global, per-handle setting. Does it make sense to have a per-operation override as can be done for other configuration?

Reply:
I m not sure, but i don't see much use case for this.  Not all config settings are overridable per operation, e.g. dbNumber setting is also config only (this is for use of 3rd party library for db number type).  There are several other settings that are config-only as well.  So i kept it the same as dbNumber.  If you do see a use case for this, i can implement it of course.